toc/rfc/rfc-draft-promoting-or-revoking-wg-leads.md

```diff
@@ -0,0 +1,51 @@
+# Meta
+[meta]: #meta
+- Name: Promoting and Revoking Working Group Leads
+- Start Date: 2023-10-11
+- Author(s): @ameowlia
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: https://github.com/cloudfoundry/community/pull/704
+
+## Summary
+
+Define the process for promoting or removing Tech Leads and Execution Leads for
+Working Groups.
+
+## Problem
+
+Currently there is no defined process for how to promote or remove Tech Leads
+or Execution Leads for Working Groups. This process should be defined.
+
+## Proposal
+In this proposal the word "Lead" encompasses both the "Tech Lead" and
+"Execution Lead" roles. As currently stated in the
+[ROLES.md](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md)
+document, both Lead roles are sponsored by the TOC. 
+
+### Promotion to Lead
+* If a person meets the criteria to be a Lead as defined in
+  [ROLES.md](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md),
+  they MAY submit a PR adding themselves to the contributors.yml as a Lead.
+
+* The TOC will start a one week final comment period on the PR to collect
+  approvals from the required parties.
+
+* The TOC MUST approve the PR before it can be merged.
+
+* At least one current Lead for the affected Working Group MUST approve the PR
```

Comment:
This can become a deadlock e.g. when there is only one WG lead who is stepping down and doesn't approve the PR for whatever reason. Well, we have [rfc-0021-toc-stand-in-for-wg-lead](https://github.com/cloudfoundry/community/blob/rfc-becoming-wg-lead/toc/rfc/rfc-0021-toc-stand-in-for-wg-lead.md) for such situations.
Maybe we change the MUST into a SHOULD.

Reply:
👍